mage.go

```diff
@@ -3,9 +3,85 @@
 package main
 
 import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+    "strings"
 	"github.com/magefile/mage/sh"
```

Comment:
there should be an empty line between standard lib imports and remote imports. this will keep them separate when you run `go fmt`

Reply:
fixed by go fmt!